src/components/Dialogs/FixedDimension.js

```diff
@@ -0,0 +1,173 @@
+}) => {
+    const selectUiItems = ({ dimensionId, items }) => {
+        setUiItems({
+            dimensionId,
+            itemIds: items.map(item => item.id),
+        })
+
+        switch (dimensionId) {
+            case DIMENSION_ID_ORGUNIT: {
+                const forMetadata = {}
+                const forParentGraphMap = {}
+
+                items.forEach(ou => {
+                    const id = ouIdHelper.removePrefix(ou.id)
+                    forMetadata[id] = {
+                        id,
+                        name: ou.name || ou.displayName,
+                        displayName: ou.displayName,
+                    }
+
+                    if (ou.path) {
+                        const path = removeLastPathSegment(ou.path)
+
+                        forParentGraphMap[ou.id] =
+                            path === `/${ou.id}` ? '' : path.replace(/^\//, '')
+                    }
+                })
+
+                addMetadata(forMetadata)
+                addParentGraphMap(forParentGraphMap)
+
+                break
+            }
+            // TODO: case DIMENSION_ID_PERIOD:
+        }
+    }
+
+    const closeModal = () => onClose()
+
+    const renderModalContent = () => {
+        switch (dimensionId) {
+            case DIMENSION_ID_ORGUNIT: {
+                const dimensionProps = {
+                    onSelect: selectUiItems,
+                }
+
+                const selected = ouIds // TODO: Refactor to not depend on the whole metadata object, but pass in full ouObjects (mapped with metadata) instead of just ids
+                    .filter(
+                        id =>
+                            metadata[ouIdHelper.removePrefix(id)] !== undefined
+                    )
+                    .map(id => {
+                        const ouUid = ouIdHelper.removePrefix(id)
+                        return {
+                            id,
+                            name:
+                                metadata[ouUid].displayName ||
+                                metadata[ouUid].name,
+                            path: getOuPath(ouUid, metadata, parentGraphMap),
+                        }
+                    })
+
+                const display =
+                    DIMENSION_ID_ORGUNIT === dimensionId ? 'block' : 'none'
+
+                return (
+                    <div key={DIMENSION_ID_ORGUNIT} style={{ display }}>
+                        <OrgUnitDimension
+                            selected={selected}
+                            roots={rootOrgUnits.map(
+                                rootOrgUnit => rootOrgUnit.id
+                            )}
+                            {...dimensionProps}
+                        />
+                    </div>
+                )
+            }
+            // TODO: case DIMENSION_ID_PERIOD:
+        }
+    }
+
+    const primaryOnClick = () => {
+        onUpdate()
+        closeModal()
+    }
+
+    const renderModalTitle = () => dimension.name
```

Comment:
You can just pass the string to `title`

Reply:
yeah, good spot, too much copy pasting on my part 😄